app/static/js/find.js

```diff
@@ -25,9 +26,11 @@ let markers = [];
       userQuery - sanitized user query string
 */
 
-// zoom to user location only if no query in URL
-if (!userQuery && !userLatLon.lat && navigator.geolocation) {
-    navigator.geolocation.getCurrentPosition(geoSuccess);
+// zoom to user location only if no query in URL and if we're not on a Destination page
+if (!userQuery && !userLatLon.lat && !isDestinationPage && navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(geoSuccess, function() {
+		alert('You did not grant permission for the app to access your location, so we cannot show you your location.')
```

Comment:
Please change to "No location specified". (the user can also specify a location using the search function)

Reply:
+1 !